### PR TITLE
TELCODOCS-1916 - Publish RDS without password

### PR DESCRIPTION
**Pull request by aireilly**
As part of decommissioning docs.openshift work, we need to bring the Telco RDS docs into the public-facing docs.

Version(s):
enterprise-4.14+

Issue:
https://issues.redhat.com/browse/TELCODOCS-1916

Link to docs previews:
* [Updates to public facing docs](https://78304--ocpdocs-pr.netlify.app/openshift-enterprise/latest/telco_ref_design_specs/ran/telco-ran-ref-design-spec)
* [Updates to previously published private openshift-telco 4.14 and 4.15 docs](https://file.emea.redhat.com/aireilly/TELCODOCS-1920/welcome/index.html)

QE review:
- [x] QE not required. Docs are unchanged, just changing location.

### CODE REVIEW

**Review by MarSik — APPROVED** [review]
/lgtm

**Comment by dfitzmau** [comment]
The link here is not working, but maybe the content needs to be merged upstream first? 

Screenshot from 2024-07-01 14-37-53

File: telco_ref_design_specs/ran/telco-ran-du-overview.adoc [hunk 1]
```diff
@@ -15,10 +15,19 @@ include::modules/telco-reference-application-workload-characteristics.adoc[level
...
 include::modules/telco-ran-hub-cluster-management.adoc[leveloffset=+1]
 
 include::modules/telco-ran-du-reference-components.adoc[leveloffset=+1]
 
-include::modules/telco-ran-measured-kpi-results.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+
+* For details of the telco RAN RDS KPI test results, see link:https://access.redhat.com/articles/7076937[Telco RAN RDS KPI test results overview].
+This informtion is only avaaible to customers and partners.
```
Comment (dfitzmau) [inline telco_ref_design_specs/ran/telco-ran-du-overview.adoc:31]:
```suggestion
This information is only available to customers and partners.
```

**Comment by aireilly** [comment]
> The link here is not working, but maybe the content needs to be merged upstream first?
> 
> Screenshot from 2024-07-01 14-37-53

Yes needs to be merged before the link is made valid

File: telco_ref_design_specs/ran/telco-ran-du-overview.adoc [hunk 2]
```diff
@@ -15,10 +15,19 @@ include::modules/telco-reference-application-workload-characteristics.adoc[level
...
+
 include::modules/telco-ran-hub-cluster-management.adoc[leveloffset=+1]
 
 include::modules/telco-ran-du-reference-components.adoc[leveloffset=+1]
 
-include::modules/telco-ran-measured-kpi-results.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+
+* For details of the telco RAN RDS KPI test results, see link:https://access.redhat.com/articles/7076937[Telco RAN RDS KPI test results overview].
```
Comment (dfitzmau) [inline telco_ref_design_specs/ran/telco-ran-du-overview.adoc:30]:
```suggestion
* For details of the telco RAN RDS KPI test results, see link:https://access.redhat.com/articles/7076937[Telco RAN RDS KPI test results].
```

To match the article name 100%

File: welcome/index.adoc [hunk 3]
```diff
@@ -40,14 +40,14 @@ endif::[]
 
 ifdef::openshift-telco[]
 [.lead]
-This documentation describes the telco core and telco RAN DU reference design specifications (RDS).
 
 [IMPORTANT]
 ====
-This content is shared with Red Hat telco partners only.
+The telco core and telco RAN DU reference design specifications (RDS) are no longer published at this location.
```
Comment (dfitzmau) [inline welcome/index.adoc:46]:
```suggestion
The Telco core and Telco RAN distributed unit (DU) reference design specifications (RDS) are no longer published at this location.
```

File: welcome/index.adoc [hunk 4]
```diff
@@ -40,14 +40,14 @@ endif::[]
...
+
+For the latest version of the telco RDS, see link:https://docs.openshift.com/container-platform/4.15/telco_ref_design_specs/telco-ref-design-specs-overview.html[Telco core and RAN DU reference design specifications].
```
Comment (dfitzmau) [inline welcome/index.adoc:48]: Ah, yes, this link does not render for me:

Screenshot from 2024-07-01 14-50-34

**Comment by dfitzmau** [comment]
Hi @aireilly . Nice work with moving content, buy no means a simple day in the sun. I added comments inline. The broken link would be the biggie to fix unless it cannot be tested until pushing to the upstream branch.

File: welcome/index.adoc (hunk 3 above)
Comment (aireilly) [reply welcome/index.adoc:46]: No "telco" doesn't take caps

**Comment by aireilly** [comment]
> Hi @aireilly . Nice work with moving content, buy no means a simple day in the sun. I added comments inline. The broken link would be the biggie to fix unless it cannot be tested until pushing to the upstream branch.

Thanks - yep link will be broken until we merge and the docs go live

**Comment by dfitzmau** [comment]
I found some more broken links with existing content that might vbe broken because of the restructuring: 

Screenshot from 2024-07-01 14-52-49
Screenshot from 2024-07-01 14-53-50
Screenshot from 2024-07-01 14-55-40

**Comment by aireilly** [comment]
> I found some more broken links with existing content that might vbe broken because of the restructuring:

Oh good catch! Need to update all these links. Thanks!

**Comment by dfitzmau** [comment]
Thanks, Aidan. Please let me know when they are fixed and I'll take another look.

**Review by dfitzmau — APPROVED** [review]

**Pull request merged**